File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.java
import java.util.ArrayList;

class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        ArrayList<Integer> answer = new ArrayList<>();
        for(String str : intStrs){
            int num = Integer.parseInt(str.substring(s,s+l));
            if(num > k){
                answer.add(num);
            }
        }
        return answer.stream().mapToInt(i -> i).toArray();
    }
}

File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.js
function solution(intStrs, k, s, l) {
    let answer = [];
    intStrs.map(function(str,idx){        
        let num = parseInt(str.substring(s, s + l));
        if(num > k){
            answer.push(num); 
        }
    }); 
    return answer;
}
